File: AD-HOC/2315/main.c
#include <stdio.h>

int elapsed(int fromDay, int fromMonth, int toDay, int toMonth) {
    int days[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    int total = toDay - fromDay;
    for (int month = fromMonth; month < toMonth; month++)
        total += days[month - 1];
    return total;
}

int main() {
    int fromDay, fromMonth, toDay, toMonth;
    scanf("%d %d %d %d", &fromDay, &fromMonth, &toDay, &toMonth);
    printf("%d\n", elapsed(fromDay, fromMonth, toDay, toMonth));
    return 0;
}

File: AD-HOC/2315/Main.java
import java.util.Map;
import java.util.Scanner;

public class Main {

    static int elapsed(int fromDay, int fromMonth, int toDay, int toMonth) {
        Map<Integer, Integer> days = Map.ofEntries(Map.entry(1, 31), Map.entry(2, 28), Map.entry(3, 31),
                Map.entry(4, 30), Map.entry(5, 31), Map.entry(6, 30), Map.entry(7, 31), Map.entry(8, 31),
                Map.entry(9, 30), Map.entry(10, 31), Map.entry(11, 30), Map.entry(12, 31));
        int total = toDay - fromDay;
        for (int month = fromMonth; month < toMonth; month++)
            total += days.get(month);
        return total;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int fromDay = scanner.nextInt();
        int fromMonth = scanner.nextInt();
        int toDay = scanner.nextInt();
        int toMonth = scanner.nextInt();
        System.out.println(elapsed(fromDay, fromMonth, toDay, toMonth));
        scanner.close();
    }
}
